packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +126,31 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  
+  all: function() {
+    this.allDeps.depend();
+    var ret = {};
+    _.each(this.keys, function(value, key) {
+      ret[key] = parse(value);
+    });
+    return ret;
+  },
+  
+  clear: function() {
+    var self = this;
+    
+    var oldKeys = self.keys;
+    self.keys = {};
+    
+    self.allDeps.changed();
+    
+    _.each(oldKeys, function(value, key) {
```

Comment:
Yeah, that's what I was thinking too. 

Actually, is this an unnecessary optimization? `clear()` is just going to call `changed()` a *bunch* of times on different dependencies, which will in turn invalidate a bunch of computations, but nothing will actually happen until the next flush cycle. So in terms of reruns there'll be no difference, won't there?

Happy to make the change if you want.